examples/visual-tests/amp-story-player/back-button.html

```diff
@@ -0,0 +1,54 @@
+<html>
+  <head>
+    <title>Story Player (Non-AMP)</title>
+    <script>
+      // TODO(): Remove this when unit tests for player are fixed.
+      __AMP_VISUAL_TEST = true;
+    </script>
+    <script async src="../../../dist/amp-story-player.js"></script>
+    <link href="../../../build/css/amp-story-player-v0.css" rel='stylesheet' type='text/css'>
+    <style page>
+      body {
+        font-family: sans-serif;
+      }
+    </style>
+    <style injected>
+      .story-player-iframe {
+        background-size: cover;
+        display: block;
+        position: relative;
+        height: 100%;
+        width: 100%;
+        flex: 0 0 100%;
+      }
+
+      .title {
+        background: linear-gradient(to bottom, rgba(0, 0, 0, 0), rgba(0, 0, 0, 1));
+        display: block;
+        position: absolute;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        color: #fff;
+        font-size: 40px;
+        padding: 88px 32px 32px 32px;
+        line-height: 1.25;
+        text-shadow: 3px 3px #000;
+      }
+
+      /* TODO: Remove this when poster image is no longer displayed. */
+      amp-story-player a:first-of-type {
+        display: none;
+      }
+    </style>
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  </head>
+  <body>
+    <amp-story-player style="width: 360px; height: 600px;" exit-control="back-button">
+      <a href="../amp-story/amp-story-grid-layer-template-fill.html"
```

Comment:
using a local story in this form or `href="/examples/visual-tests/...` yields the follwing HTML which doesn't populate the document inside the `<iframe>`
```

  | <!DOCTYPE html><html><head>
-- | --
  | <title>Story Player (Non-AMP)</title>
  | <script>
  | // TODO(): Remove this when unit tests for player are fixed.
  | __AMP_VISUAL_TEST = true;
  | </script>
  |  
  | <link href="../../../build/css/amp-story-player-v0.css" rel="stylesheet" type="text/css">
  | <style page="">
  | body {
  | font-family: sans-serif;
  | }
  | </style>
  | <style injected="">
  | .story-player-iframe {
  | background-size: cover;
  | display: block;
  | position: relative;
  | height: 100%;
  | width: 100%;
  | flex: 0 0 100%;
  | }
  |  
  | .title {
  | background: linear-gradient(to bottom, rgba(0, 0, 0, 0), rgba(0, 0, 0, 1));
  | display: block;
  | position: absolute;
  | left: 0;
  | right: 0;
  | bottom: 0;
  | color: #fff;
  | font-size: 40px;
  | padding: 88px 32px 32px 32px;
  | line-height: 1.25;
  | text-shadow: 3px 3px #000;
  | }
  |  
  | /* TODO: Remove this when poster image is no longer displayed. */
  | amp-story-player a:first-of-type {
  | display: none;
  | }
  | </style>
  | <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
  | <script type="">window.PercyAgent=function(e){var t={};function n(r){if(t[r])return t[r].exports;var o=t[r]={i:r,l:!1,exports:{}};return e[r].call(o.exports,o,o.exports,n),o.l=!0,o.exports}return n.m=e,n.c=t,n.d=function(e,t,r){n.o(e,t)\|\|Object.defineProperty(e,t,{enumerable:!0,get:r})},n.r=function(e){"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},n.t=function(e,t){if(1&t&&(e=n(e)),8&t)return e;if(4&t&&"object"==typeof e&&e&&e.__esModule)return e;var r=Object.create(null);if(n.r(r),Object.defineProperty(r,"default",{enumerable:!0,value:e}),2&t&&"string"!=typeof e)for(var o in e)n.d(r,o,function(t){return e[t]}.bind(null,o));return r},n.n=function(e){var t=e&&e.__esModule?function(){return e.default}:function(){return e};return n.d(t,"a",t),t},n.o=function(e,t){return Object.prototype.hasOwnProperty.call(e,t)},n.p="",n(n.s=0)}([function(e,t,n){"use strict";n.r(t);var r=5338;function o(e,t){for(var n=0;n<t.length;n++){var r=t[n];r.enumerable=r.enumerable\|\|!1,r.configurable=!0,"value"in r&&(r.writable=!0),Object.defineProperty(e,r.key,r)}}function i(e,t,n){return t in e?Object.defineProperty(e,t,{value:n,enumerable:!0,configurable:!0,writable:!0}):e[t]=n,e}var a=function(){function e(t,n){!function(e,t){if(!(e instanceof t))throw new TypeError("Cannot call a class as a function")}(this,e),i(this,"originalDOM",void 0),i(this,"clonedDOM",void 0),i(this,"options",void 0),i(this,"defaultDoctype","<!DOCTYPE html>"),this.originalDOM=t,this.options=n\|\|{},this.clonedDOM=this.cloneDOM()}var t,n,r;return t=e,(n=[{key:"snapshotString",value:function(){var e=this.clonedDOM,t=this.getDoctype();if(this.options.domTransformation)try{e=this.options.domTransformation(e)}catch(e){console.error("Could not transform the dom: ",e)}return t+e.outerHTML}},{key:"getDoctype",value:function(){return this.clonedDOM.doctype?this.doctypeToString(this.clonedDOM.doctype):this.defaultDoctype}},{key:"doctypeToString",value:function(e){var t=e.publicId?' PUBLIC "'.concat(e.publicId,'" '):"",n=e.systemId?' SYSTEM "'.concat(e.systemId,'" '):"";return"<!DOCTYPE ".concat(e.name)+t+n+">"}},{key:"cloneDOM",value:function(){this.mutateOriginalDOM();var e=this.originalDOM.cloneNode(!0);return e=this.stabilizeDOM(e)}},{key:"stabilizeDOM",value:function(e){return this.serializeInputElements(e),this.options.enableJavaScript\|\|this.serializeCSSOM(e),e.documentElement}},{key:"serializeInputElements",value:function(e){var t=this.originalDOM.querySelectorAll("input, textarea, select");Array.from(t).forEach(function(t){var n=t.getAttribute("data-percy-element-id"),r='[data-percy-element-id="'.concat(n,'"]'),o=e.querySelector(r);switch(t.type){case"checkbox":case"radio":t.checked&&o.setAttribute("checked","");break;case"select-one":-1!==t.selectedIndex&&o.options[t.selectedIndex].setAttribute("selected","true");break;case"select-multiple":var i=Array.from(t.selectedOptions),a=Array.from(o.options);i.length&&i.forEach(function(e){a.find(function(t){return e.text===t.text}).setAttribute("selected","true")});break;case"textarea":o.innerHTML=t.value;break;default:o.setAttribute("value",t.value)}})}},{key:"serializeCSSOM",value:function(e){Array.from(this.originalDOM.styleSheets).forEach(function(t){if(o=t.href,i=t.ownerNode,a=i.innerText&&i.innerText.length>0,!o&&!a&&t.cssRules){var n=e.createElement("style"),r=Array.from(t.cssRules).reduce(function(e,t){return e+t.cssText},"");n.type="text/css",n.setAttribute("data-percy-cssom-serialized","true"),n.innerHTML=r,e.head.appendChild(n)}var o,i,a})}},{key:"mutateOriginalDOM",value:function(){var e=this.originalDOM.querySelectorAll("input, textarea, select");Array.from(e).forEach(function(e){var t,n;e.getAttribute("data-percy-element-id")\|\|(t=e,n="_".concat(Math.random().toString(36).substr(2,9)),t.setAttribute("data-percy-element-id",n))})}}])&&o(t.prototype,n),r&&o(t,r),e}();function c(e,t){for(var n=0;n<t.length;n++){var r=t[n];r.enumerable=r.enumerable\|\|!1,r.configurable=!0,"value"in r&&(r.writable=!0),Object.defineProperty(e,r.key,r)}}function s(e,t,n){return t in e?Object.defineProperty(e,t,{value:n,enumerable:!0,configurable:!0,writable:!0}):e[t]=n,e}var l=function(){function e(t,n){!function(e,t){if(!(e instanceof t))throw new TypeError("Cannot call a class as a function")}(this,e),s(this,"xhr",void 0),s(this,"agentHost",void 0),s(this,"agentConnected",!1),this.agentHost=t,this.xhr=new n\|\|new XMLHttpRequest,this.healthCheck()}var t,n,r;return t=e,(n=[{key:"post",value:function(e,t){this.agentConnected?(this.xhr.open("post","".concat(this.agentHost).concat(e),!1),this.xhr.setRequestHeader("Content-Type","application/json"),this.xhr.send(JSON.stringify(t))):console.warn("percy agent not started.")}},{key:"healthCheck",value:function(){var e=this;try{this.xhr.open("get","".concat(this.agentHost).concat("/percy/healthcheck"),!1),this.xhr.onload=function(){200===e.xhr.status&&(e.agentConnected=!0)},this.xhr.send()}catch(e){this.agentConnected=!1}}}])&&c(t.prototype,n),r&&c(t,r),e}();function u(e,t){var n=Object.keys(e);if(Object.getOwnPropertySymbols){var r=Object.getOwnPropertySymbols(e);t&&(r=r.filter(function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable})),n.push.apply(n,r)}return n}function h(e){for(var t=1;t<arguments.length;t++){var n=null!=arguments[t]?arguments[t]:{};t%2?u(n,!0).forEach(function(t){d(e,t,n[t])}):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(n)):u(n).forEach(function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(n,t))})}return e}function f(e,t){for(var n=0;n<t.length;n++){var r=t[n];r.enumerable=r.enumerable\|\|!1,r.configurable=!0,"value"in r&&(r.writable=!0),Object.defineProperty(e,r.key,r)}}function d(e,t,n){return t in e?Object.defineProperty(e,t,{value:n,enumerable:!0,configurable:!0,writable:!0}):e[t]=n,e}var p=function(){function e(){var t=arguments.length>0&&void 0!==arguments[0]?arguments[0]:{};!function(e,t){if(!(e instanceof t))throw new TypeError("Cannot call a class as a function")}(this,e),d(this,"clientInfo",void 0),d(this,"environmentInfo",void 0),d(this,"xhr",void 0),d(this,"handleAgentCommunication",void 0),d(this,"port",void 0),d(this,"domTransformation",void 0),d(this,"client",null),this.clientInfo=t.clientInfo\|\|null,this.environmentInfo=t.environmentInfo\|\|null,this.handleAgentCommunication=!1!==t.handleAgentCommunication,this.domTransformation=t.domTransformation\|\|null,this.port=t.port\|\|r,this.handleAgentCommunication&&(this.xhr=t.xhr\|\|XMLHttpRequest,this.client=new l("http://localhost:".concat(this.port),this.xhr))}var t,n,o;return t=e,(n=[{key:"snapshot",value:function(e){var t=arguments.length>1&&void 0!==arguments[1]?arguments[1]:{},n=t.document\|\|document,r=this.domSnapshot(n,t);return this.handleAgentCommunication&&this.client&&this.client.post("/percy/snapshot",{name:e,url:n.URL,percyCSS:t.percyCSS,enableJavaScript:t.enableJavaScript\|\|t.enableJavascript,widths:t.widths,minHeight:t.minHeight\|\|t.minimumHeight,clientInfo:this.clientInfo,environmentInfo:this.environmentInfo,domSnapshot:r}),r}},{key:"domSnapshot",value:function(e){var t=arguments.length>1&&void 0!==arguments[1]?arguments[1]:{},n=new a(e,h({},t,{domTransformation:this.domTransformation}));return n.snapshotString()}}])&&f(t.prototype,n),o&&f(t,o),e}();t.default=p}]).default;//# sourceURL=/Users/artezan/amphtml/build-system/tasks/visual-diff/node_modules/@percy/puppeteer/node_modules/@percy/agent/dist/public/percy-agent.js</script></head>
  | <body>
  | <amp-story-player style="width: 360px; height: 600px;" exit-control="back-button" class="i-amphtml-story-player-loaded i-amphtml-story-loaded">
  | <a href="/examples/visual-tests/amp-story/amp-story-grid-layer-template-fill.html" class="story">
  | <span class="title">Test one</span>
  | </a>
  | <style>:host{all:initial;display:block;border-radius:0!important;width:360px;height:600px;overflow:auto}.story-player-iframe{height:100%;width:100%;-ms-flex:0 0 100%;flex:0 0 100%;border:0;opacity:0;transition:opacity 500ms ease;position:absolute}main{height:100%;position:relative;overflow:hidden;transform-style:preserve-3d}.i-amphtml-story-player-loaded iframe{opacity:1;transition:transform 200ms cubic-bezier(0.4,0,0.2,1)}.story-player-iframe:first-child,main .story-player-iframe[i-amphtml-iframe-position="0"]{transform:translateZ(1px)}.story-player-iframe:nth-child(2),.story-player-iframe:nth-child(3),main .story-player-iframe[i-amphtml-iframe-position="1"]{transform:translate3d(100%,0,0)}main .story-player-iframe[i-amphtml-iframe-position="-1"]{transform:translate3d(-100%,0,0)}.amp-story-player-back-button,.amp-story-player-close-button{position:absolute;height:48px;width:48px;background-repeat:no-repeat;background-position:50%;margin-top:7px;background-size:28px;border:0px;background-color:transparent;outline:transparent}.amp-story-player-close-button{background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' height='24' width='24' fill='%23FFF'%3E%3Cpath d='M0 0h24v24H0z' fill='none'/%3E%3Cpath d='M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z'/%3E%3C/svg%3E")}.amp-story-player-back-button{background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' height='24' width='24' fill='%23FFF'%3E%3Cpath d='M0 0h24v24H0z' fill='none'/%3E%3Cpath d='M20 11H7.83l5.59-5.59L12 4l-8 8 8 8 1.41-1.41L7.83 13H20v-2z'/%3E%3C/svg%3E")}
  | /*# sourceURL=/css/amp-story-player-iframe.css*/</style><main class="i-amphtml-story-player-loaded"><iframe class="story-player-iframe" sandbox="allow-top-navigation-by-user-activation allow-popups-to-escape-sandbox allow-forms allow-modals allow-pointer-lock allow-popups allow-same-origin allow-scripts allow-top-navigation" src="http://0.localhost:8000/examples/visual-tests/amp-story/amp-story-grid-layer-template-fill.html?amp_js_v=0.1#visibilityState=visible&amp;origin=http%3A%2F%2F0.localhost%3A8000&amp;showStoryUrlInfo=0&amp;storyPlayer=v0&amp;cap=swipe" title="Test one"></iframe><button class="amp-story-player-back-button"></button></main></amp-story-player>
  |  
  |  
  | </body></html>


```